src/UploadType.php

```diff
@@ -22,16 +22,21 @@ class UploadType extends ScalarType
     /**
      * @var string
      */
-    public $description
-        = 'The `Upload` special type represents a file to be uploaded in the same HTTP request as specified by
- [graphql-multipart-request-spec](https://github.com/jaydenseric/graphql-multipart-request-spec).';
+    public $description;
+
+    public function __construct(array $config = [])
+    {
+        parent::__construct($config);
+
+        $this->description = 'The `' . $this->name . '` special type represents a file to be uploaded in the same HTTP request as specified by [graphql-multipart-request-spec](https://github.com/jaydenseric/graphql-multipart-request-spec).';
```

Comment:
You are overwriting the potentially custom description given in `$config`. So by trying to allow customizing the name, you don't allow to customize the description anymore. That should not be the case.